include/megaapi_impl.h

```diff
@@ -2734,6 +2735,7 @@ class MegaApiImpl : public MegaApp
         virtual void chatlinkurl_result(handle, int, string*, string*, int, m_time_t, error);
         virtual void chatlinkclose_result(error);
         virtual void chatlinkjoin_result(error);
+        virtual void folderlinkinfo_result(error, handle , handle, string*, string* , m_off_t, uint32_t , uint32_t , m_off_t , uint32_t);
```

Comment:
This method is not specific of chat, despite it's first usage will be for chats. It shouldn't be inside the `ENABLE_CHAT` ifdefs